n_queen_problem/n_queen_problem.go

```diff
@@ -0,0 +1,67 @@
+// Given a chess board having N×N cells
+// place N queens on the board in such a way that no queen attacks any other queen.
+
+// There can be a number of possible solutions for a given length of board.
+// This implementation prints only one valid solution, it can be extended to print all possible valid solutions.
+
+package main
+
+import "fmt"
+
+func main() {
+	n := 8 // size of chess board (n x n)
+
+	// make board
+	b := make([][]int, n)
```

Comment:
Please use common variable names such as, n => size, b => board.

Reply:
Yes that makes more sense. Done.